forge/db/controllers/ProjectTemplate.js

```diff
@@ -12,29 +88,56 @@ module.exports = {
                     if (httpAdminRoot[httpAdminRoot.length - 1] === '/') {
                         httpAdminRoot = httpAdminRoot.substring(0, httpAdminRoot.length - 1)
                     }
```

Comment:
What happens if `httpAdminRoot` is entered as `/`? The substring call makes it an empty string

Reply:
Then an empty string gets passed down which is the same as not setting it - I'm 99% sure NR does the right thing with that.

Reply:
yeah, it would evaluate to false